classes/class-wc-connect-shipping-method.php

```diff
@@ -437,7 +437,8 @@ public function calculate_shipping( $package = array() ) {
 						'cost'      => $rate->rate,
 						'meta_data' => array(
 							'wc_connect_packages' => $rate->packages,
-							__( 'Packaging', 'woocommerce-services' ) => $packaging_info
+							__( 'Packaging', 'woocommerce-services' ) => $packaging_info,
+							'wc_connect_packing_log' => $rate->box_packing_log,
```

Comment:
Can we also handle the case where `$rate->box_packing_log` is NULL? (i.e. the server does not respond with a `box_packing_log` property)

Reply:
Done in f9865626.